crates/rnote-ui/src/canvas/input.rs

```diff
@@ -119,7 +119,11 @@ pub(crate) fn handle_pointer_controller_event(
                 if gdk_button == gdk::BUTTON_PRIMARY {
                     pen_state = PenState::Up;
                 } else {
-                    pen_state = PenState::Proximity;
+                    if cfg!(target_os = "windows") {
```

Comment:
A code comment here with the reason for this windows-specific quirk would be nice (and I see that clippy will complain about a collapsible if/else, you can turn it off with `#[allow(clippy::collapsible_else_if)]`

Reply:
fixed